src/leaderboard.js

```diff
@@ -0,0 +1,23 @@
+const { redis } = require('./redis');
+
+const addTeam = (org, score, name) => {
+    redis.zadd([org, score, name])
+        .then(console.log)
+        .catch(console.error);
+}
+
+const updateTeamScore = (org, score, name) => {
+    redis.zincrby([org, score, name])
+        .then(console.log)
+        .then(console.error)
+}
+
+const getSortedTeamsByScore = (org) => {
+    redis.zrevrange([org, 0, -1])
+        .then(console.log)
+        .then(console.error)
+}
+
```

Comment:
How are these tested? especially the getSortedTeamsByScore?

Reply:
Currently no unit tests can be written since we need to integrate with a db. However they have been verified manually.